.editorconfig

```diff
@@ -0,0 +1,13 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
Even this is related to your editor. So highly feel even this should be added to `.gitignore`